src/DynamoCoreWpf/Rendering/HelixRenderPackage.cs

```diff
@@ -57,10 +57,95 @@ public HelixRenderPackage()
             lines = InitLineGeometry();
             mesh = InitMeshGeometry();
             lineStripVertexCounts = new List<int>();
+            Transform = System.Windows.Media.Media3D.Matrix3D.Identity.ToArray();
         }
 
         #endregion
 
+
+        #region ITransformable implementation
+        /// <summary>
+        /// sets the transform that will be applied to all geometry in the renderPackage
+        /// </summary>
+        /// <param name="transform"></param>
+        public void SetTransform(Autodesk.DesignScript.Geometry.CoordinateSystem transform)
+        {
+            var xaxis = transform.XAxis;
+            var yaxis = transform.YAxis;
+            var zaxis = transform.ZAxis;
+            var org = transform.Origin;
+
+            var csAsMat = new System.Windows.Media.Media3D.Matrix3D(xaxis.X, xaxis.Z, -xaxis.Y, 0,
+                                                                    zaxis.X, zaxis.Z, -zaxis.Y, 0,
+                                                                    -yaxis.X, -yaxis.Z, yaxis.Y, 0,
+                                                                      org.X, org.Z, -org.Y, 1);
+
+
+            this.Transform = csAsMat.ToArray();
+        }
+
+        /// <summary>
+        /// sets the transform that will be applied to all geometry in the renderPackage
+        /// by computing the matrix that transforms between from and to
+        /// </summary>
+        /// <param name="from"></param>
+        /// <param name="to"></param>
+        public void SetTransform(Autodesk.DesignScript.Geometry.CoordinateSystem from, Autodesk.DesignScript.Geometry.CoordinateSystem to)
+        {
+            var inverse = from.Inverse();
+            var final = inverse.PreMultiplyBy(to);
+
+            this.SetTransform(final);
+        }
+
+
+        /// <summary>
+        /// sets the transform that will be applied to all geometry in the renderPackage
+        /// as this is a helix specific implementation it should be noted that the this matrix
+        /// should be as follows when converting from a ProtoGeometry/Dynamo CoordinateSystem
+        /// [ xaxis.X, xaxis.Z, -xaxis.Y, 0,
+        /// zaxis.X, zaxis.Z, -zaxis.Y, 0,
+        /// -yaxis.X, -yaxis.Z, yaxis.Y, 0,
+        /// org.X, org.Z, -org.Y, 1 ]
+        /// as Helix and Dynamo have their Y and Z axes reversed
```

Comment:
Ugh. We really need to fix this. I realize it's outside the scope of this PR, but we need to just push verts into render packages with Y up then add a transform to the entire scene to flip it the right way around so we never have to do things conditionally. Then this comment could just say that "transforms are row-ordered with +Y up".